### ISSUE TITLE
Directory listing page not accessible

### ISSUE THREAD

**Issue by dajoturner** [ucsc/wcms-design-templates#266]
Len Burns submitted this issue. Setting as Critical and assigning to Jeff for assessment. @jrosczyk If this requires changes to the PHP feed, we probably need to set it to Could Wait or even Fix Later. 

See: http://history-dev.ucsc.edu/faculty/profiles/index.php

Len's text:
The structure of this page is confusing.  First, everything is at a level 1 heading except the listed names of the faculty which are at level 3.  From a semantic point of view, this is confusing since many users of adaptive technologies explore by heading.  The format of each faculty member record is also confusing.  It is using a table, I am guessing as a layout structure.  When I go to the level 3 heading of a faculty member, the screen reader is not informing me that I am now within a table.  That is because it is jumping over the point at which the table begins.  This is problematic since the keystrokes I would use to navigate within the table differ from thos normally used.
Last but not least, the navigation allowing me to select “All Faculty,” “Regular Faculty,” etc, I nearly missed completely.  This is because it is located just after the main region and is not identified by any markup.  From my perspective, it is nearly at the bottom of the page.  Were I not scrutinizing the page, I would have had no indication of the existence of these controls.
